frontend/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
@hoob3rt react generuje tego enva, ale nie było go wcześniej w repo. To jest coś co powinno trafić czy powinniśmy to olać?

Reply:
to jest trash, wgle tego nie potrzebujemy. `create-react-app` go generuje. @WojtAcht do usuniecia